embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/opentelemetry/EmbraceAttributeKeys.kt

```diff
@@ -2,11 +2,21 @@ package io.embrace.android.embracesdk.opentelemetry
 
 import io.embrace.android.embracesdk.arch.schema.EmbraceAttributeKey
 
+/**
+ * A snapshot of the current call stack of the threads running in the app process per [ThreadInfo]
+ */
+internal val embAndroidThreads = EmbraceAttributeKey("android.threads")
+
 /**
  * Attribute name for the app framework for which the telemetry is being logged
  */
 internal val embAppFramework = EmbraceAttributeKey("app_framework")
 
+/**
+ * Sequence number for the number of crashes captured by Embrace on the device, reported on every crash
+ */
+internal val embCrashNumber = EmbraceAttributeKey("crash_number")
```

Comment:
Would be better to name it `android.crash_number` to distinguish it from other platforms.

Reply:
Does iOS have this? If so, we should share it. If not, I'll change it.